analyzer/codechecker_analyzer/analyzer_context.py

```diff
@@ -408,3 +547,6 @@ def get_context():
         import traceback
         traceback.print_exc()
         sys.exit(1)
+    except ValueError as err:
```

Comment:
I think this error handling could be moved up more, but in a later refactoring because the KeyError should be handled too there so the sys.exit calls are only at the main scripts. Probably as part of the upcoming sys.exit refactoring.

Reply:
Yes, I thought of moving this exception catching somewhere outside. Due to lack of time let me postpone this for another PR, because it would be a bigger task, since at each construction of this `Cotext` object some action is needed. Not to mention that is some modules this object is used for several different purposes and not all of them need to exit in case of a format error. On my defense let me say that currently we have many `sys.exit(1)` calls in non-toplevel scripts besides this, so it is not an exceptional place in this regard. On the other hand I'm not sure that we need to prohibit emitting status 1 from non-toplevel scripts, because these are referring CodeChecker errors either way which we want to indicate with value 1. But let this be the discussion of another PR in which we possibly tidy up every `sys.exit(1)` calls.